app/services/prison_offender_manager_service.rb

```diff
@@ -23,6 +23,9 @@ def self.get_poms(prison)
   def self.get_pom(caseload, nomis_staff_id)
```

Comment:
Not for now, but we should try and optimise this at some point.

Reply:
Agreed!